dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -128,21 +131,9 @@ public DependencyGraphBuilder(Iterable<String> mavenRepositoryUrls) {
     this.repositories = repositoryListBuilder.build();
   }
 
-  private DependencyNode resolveCompileTimeDependencies(DependencyNode root)
-      throws DependencyCollectionException, DependencyResolutionException {
-    return resolveCompileTimeDependencies(root, false);
```

Comment:
Removing unnecessary override.

Reply:
What was being overridden? Or did you mean overload?
Also, is this removal dependent on this PR or could it be done separately?

Reply:
Overload, yes.

Reply:
Reverted irrelevant refactoring.